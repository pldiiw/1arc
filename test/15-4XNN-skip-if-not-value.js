'use strict';

let engine = require('engine').initialize();
const skipIfNotValue = require('instructions').skipIfNotValue;

engine.data[7] = 143;
engine.pc = 200;

const engine1 = skipIfNotValue(engine, 7, 143);
const engine2 = skipIfNotValue(engine1, 7, 32);

console.log(engine.pc)
console.log(engine1.pc)
console.log(engine2.pc)
