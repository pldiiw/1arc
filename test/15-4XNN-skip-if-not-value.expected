200
200
202
